R/logging.R

```diff
@@ -187,8 +187,12 @@ tune_catalog <- function(issues) {
 
       # construct issue summary
       color <- if (current$type == "warning") {cli::col_yellow} else {cli::col_red}
```

Comment:
This is not part of the changes, but it is in the same code chunk so I hope you are okay with this change. I find this easier to read, and it matches with our style https://style.tidyverse.org/syntax.html#inline-statements

```suggestion
      color <- if (current$type == "warning") cli::col_yellow else cli::col_red
```